ibm_was/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "display_name": "IBM WAS",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "ibm_was",
+  "metric_prefix": "ibmwas.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
```

Comment:
Fill this out, too

Reply:
Good catch, missed this. Thanks, I completed it now.